examples/ddp_00.py

```diff
@@ -0,0 +1,50 @@
+from dataclasses import dataclass, field
+import logging
+import os
+from typing import Any, List
+
+import hydra
+from hydra.core.config_store import ConfigStore
+from omegaconf import DictConfig, MISSING
+import torch
+import torch.distributed as dist
+
+log = logging.getLogger(__name__)
+
+
+defaults = [
+    {"hydra/launcher": "joblib"}
+]
+
+@dataclass
+class DDPConf:
```

Comment:
If we plan to use general structured configs like this one often, we should probably provide them somewhere in this repo. Although they're not configen projects, we could still have a directory for them with a namespace package to import from. Perhaps `hydra-configs-torchdistributed` or `-torchtools`?

Alternatively, maybe there are some classes in [torch.distributed](https://github.com/pytorch/pytorch/tree/master/torch/distributed) we could include in `hydra-configs-torch`.

@briankosw Could you take a look and see if there are any classes in there that make sense for generating confs from?

Reply:
I'll look into it. I feel like most of PyTorch's distributed processing API aren't user-facing, but there are certain ones that come up over and over again and it could be useful to have generated configs for them.

Reply:
I would defer any attempts for creating reusable distributed related configs to after we have a solid understanding what we can support and how.

Reply:
I agree. Just something to keep in mind as we move forward.